doc/src/modules/physics/mechanics/examples/multi_degree_freedom_holonomic_system.rst

```diff
@@ -0,0 +1,99 @@
+=========================================
+Multi Degree of Freedom Holonomic System
+=========================================
```

Comment:
This example needs a figure.

Reply:
The same figure used in actual example in pydy?

Reply:
It could be, but a nice SVG figure for SymPy is the ideal solution.

Reply:
Added the same image as in  pydy